go.mod

```diff
@@ -36,7 +36,7 @@ require (
 	github.com/stretchr/testify v1.8.1
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
-	go.mongodb.org/atlas v0.22.0
+	go.mongodb.org/atlas v0.21.1-0.20230213105757-80a065fea80c
```

Comment:
This looks like downgrade but in fact it does uses the latest version of the client

Reply:
Could you clarify why? I see that the latest one is v0.22.0 https://github.com/mongodb/go-client-mongodb-atlas/releases

![Screenshot 2023-02-14 at 09 16 01](https://user-images.githubusercontent.com/5663078/218691560-1eb8ed44-f013-4875-88e4-29cabbe1aaa7.png)

Reply:
if I do `go get -u go.mongodb.org/atlas@master` I get `go.mongodb.org/atlas v0.22.1-0.20230203154520-61aa70417b31` this probably means your branch if out of date with master, ie it doesn't include the commit that was tagged as 0.22.0, can you make sure the preview branch is up to date with master, this will be a recurring task

Reply:
I will merge the master branch into sdkv2.